### PR TITLE
added new kafka topic and support to send video analytics fps

### PR DESCRIPTION
**Pull request by thanajirao**

### CODE REVIEW

**Comment by QuPengfei** [comment]
@thanajirao . 
would you take the following info as the message template? or add more info in that.
{
"time":datetime.datetime.utcnow().isoformat(),
"machine": machine,
"fps": fps,
}

File: ad-insertion/video-analytics-service/feeder/main.py [hunk 1]
```diff
@@ -114,10 +124,11 @@ def process_stream(streamstring):
                 status, fps = get_analytic_status(instanceid.strip(), pipeline)
                 print("VA feeder: segment status : " + status, flush=True)
                 if status == 'COMPLETED':
-                    print("VA feeder: Avg fps: " + str(fps), flush=True)
+                    send_video_analytics_fps(fps)
```
Comment (nnshah1) [inline ad-insertion/video-analytics-service/feeder/main.py:132]: can we add debug log with the status,fps - was useful to have

**Review by nnshah1 — COMMENTED** [review]
approved with pengfei's requests

File: ad-insertion/video-analytics-service/feeder/main.py (hunk 1 above)
Comment (thanajirao) [reply ad-insertion/video-analytics-service/feeder/main.py:132]: for every send kafka message, they have print statement in message.py.

**Comment by thanajirao** [comment]
> @thanajirao .
> would you take the following info as the message template? or add more info in that.
> {
> "time":datetime.datetime.utcnow().isoformat(),
> "machine": machine,
> "fps": fps,
> }

@QuPengfei I can do time and fps, what is machine here ?, what other information you want me to add ?

**Comment by QuPengfei** [comment]
please refer to https://github.com/OpenVisualCloud/Ad-Insertion-Sample/blob/master/script/workload.py

You can find how to get the machine name. this key will help identify multi VA service instance id.

**Comment by thanajirao** [comment]
> please refer to https://github.com/OpenVisualCloud/Ad-Insertion-Sample/blob/master/script/workload.py
> 
> You can find how to get the machine name. this key will help identify multi VA service instance id.
Updated

**Pull request merged**